plugins/org.eclipse.n4js.generator.headless/src/org/eclipse/n4js/generator/headless/HeadlessHelper.java

```diff
@@ -282,10 +282,8 @@ private boolean skipRegistering(String projectName, FileURI projectLocation,
 					+ "]. Already registered project at " + registeredProjects.get(projectName)
 					+ ", trying to register project at " + projectLocation + ".");
 
-		// duplicate is in new location, so new project with the same name -> stop compilation
-		throw new N4JSCompileException("Duplicate project id [" + projectName
-				+ "]. Already registered project at " + registeredProjectLocation
-				+ ", trying to register project at " + projectLocation + ".");
+		// Shadowing of projects with the same name. Removed with GH-1314
+		return true;
```

Comment:
I don't understand this comment. Could you explain a bit more? Also, do you mean "_Will_ be removed with GH-1314" ?

Also, should this change in behavior be reflected in the API doc of this method?